src/cli.c

```diff
@@ -70,6 +98,224 @@ static inline void noop() {};
 
 #define CLI_VERSION "libstorj-2.0.0-beta2"
 
+static int check_file_path(char *file_path)
+{
+    struct stat sb;
+
+    if (stat(file_path, &sb) == -1)
```

Comment:
Currently the style for brackets is on the same line for everything except functions, e.g.:
```
if (stat(file_path, &sb) == -1) {
```

Reply:
fixed the braces indentation as per the suggestion

Reply:
Still looks like these are unchanged